client/my-sites/plugins/categories/use-categories.tsx

```diff
@@ -48,7 +48,7 @@ export function useCategories(
 	}
 
 	const categories = {
-		discover: { name: __( 'Discover' ), slug: 'discover', tags: [] },
+		discover: { name: __( 'All' ), slug: 'discover', tags: [] },
```

Comment:
Not sure about this change since we are not showing all the plugins, but the discover ones. I think this maybe something in the design that doesn't match production!